server/services/deliusOfficeLocationFilter.ts

```diff
@@ -0,0 +1,25 @@
+import InterventionsService from './interventionsService'
+import ReferenceDataService from './referenceDataService'
+import DeliusOfficeLocation from '../models/deliusOfficeLocation'
+
+export default class DeliusOfficeLocationFilter {
+  constructor(private referenceDataService: ReferenceDataService, private interventionService: InterventionsService) {}
```

Comment:
I wonder whether there's much gained by injecting an `InterventionsService` instance here instead of just passing an `Intervention` instance to the `findOffices` method? Less mocking to do in the tests, and potentially avoids duplicate fetches of the intervention (if we've already fetched it in the controller method).